.changeset/kind-moons-drive.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/search-select-input': major
+'@commercetools-uikit/async-select-input': major
+'@commercetools-uikit/select-input': major
+---
+
+We introduced a new (_boolean_) property (**menuIsOpen**) to Select Inputs which allows to render the input with the dropdown initially unfolded.
```

Comment:
We now have a duplicated changeset.

We should keep just one and change the versions to `minor`. (`major` is only for breaking changes).